openstudiocore/src/model/SolarCollectorIntegralCollectorStorage.cpp

```diff
@@ -189,17 +189,10 @@ namespace detail {
   bool SolarCollectorIntegralCollectorStorage_Impl::setSurface(const PlanarSurface& surface) {
     bool result(false);
 
-    if (surface.isAirWall()){
-      return false;
-    }
-
     // DLM: check for existing solar collectors or photovoltaic generators?
 
     if (surface.optionalCast<Surface>()){
-      Surface s = surface.cast<Surface>();
-      if (istringEqual("SunExposed", s.sunExposure())){
```

Comment:
I had been enforcing that solar collectors could only go on sun exposed surfaces.  However, it seemed like that might be too strict plus would require a lot of observer stuff to happen to ensure that the model never became invalid.  In the end I went for looser rules, we can always throw more warnings on translation.